napari/utils/_register.py

```diff
@@ -76,36 +93,3 @@ def create_func(cls, name=None, doc=None, filename: str = '<string>'):
     func.__doc__ = doc
 
     return func
-
-
-def _register(cls, *, name=None, doc=None):
-    from napari.components import ViewerModel
-
-    func = create_func(cls, name=name, doc=doc)
-    setattr(ViewerModel, func.__name__, func)
-    return cls
-
-
-def add_to_viewer(cls=None, *, name=None, doc=None):
-    """Adds a layer creation convenience method under viewers
-    as ``add_{name}``.
-
-    Parameters
-    ----------
-    cls : type, optional
-        Class to register.
-        If None, this function is treated as a decorator.
-    name : string, keyword-only
-        Name in snake-case of the layer name.
-        If None, is autogenerated from the class name.
-    doc : string, keyword-only
-        Docstring to use in the method.
-        If None, is autogenerated from the existing docstring.
-    """
-    if cls is not None:
-        return _register(cls, name=name, doc=doc)
-
-    def inner(cls):
-        return _register(cls, name=name, doc=doc)
-
-    return inner
```

Comment:
Why are these deleted?

Reply:
Because it is unused.

Reply:
Seems like it should be in a separate PR but ok. 👍